lib/chef_backup/helpers.rb

```diff
@@ -69,6 +69,15 @@ def log(message, level = :info)
     ChefBackup::Logger.logger.log(message, level)
   end
 
+  # For some reason, the restore codepath puts the shell_timeout flag at top
+  # level but the backup codepath puts it underneath the service_config
+  # hierarchy. Pick the one we find.
```

Comment:
This is fine if it moves us forward, by since we control the call-site as well, why not fix it there?